src/ucm/mmap/install.c

```diff
@@ -219,36 +219,37 @@ void ucm_fire_mmap_events(int events)
 }
 
 static void ucm_mmap_event_report_missing(int expected, int actual,
-                                          int is_external)
+                                          const char *event_type)
 {
     int events_count = 0;
+    int missing_events;
     int idx;
     char *buf;
     char *buf_p;
     char *end_p;
 
     UCS_STATIC_ASSERT(UCM_MMAP_REPORT_BUF_LEN <= UCS_ALLOCA_MAX_SIZE)
 
-    buf   = buf_p = ucs_alloca(UCM_MMAP_REPORT_BUF_LEN);
-    end_p = buf_p + UCM_MMAP_REPORT_BUF_LEN;
+    buf            = buf_p = ucs_alloca(UCM_MMAP_REPORT_BUF_LEN);
+    end_p          = buf_p + UCM_MMAP_REPORT_BUF_LEN;
+    missing_events = expected & ~actual & ((UCM_EVENT_VM_UNMAPPED << 1) - 1);
```

Comment:
maybe use UCS_MASK(ucs_array_size(ucm_mmap_event_name))
instead of
((UCM_EVENT_VM_UNMAPPED << 1) - 1);

Reply:
good idea :)
done